### PR TITLE
ENCM-58-sc-atac-qc-metrics-definitions

### PR DESCRIPTION
**Pull request by khzlin**

### CODE REVIEW

**Review by gabdank — DISMISSED** [review]

File: src/encoded/schemas/sc_atac_analysis_quality_metric.json [hunk 1]
```diff
@@ -136,11 +139,12 @@
...
         },
         "archr_doublet_summary_text": {
-            "title": "ArchR doublet data",
+            "title": "Pre-filtering ArchR doublet data",
+            "description": "Pre-filtering ArchR doublet data",
             "type": "object",
             "additionalProperties": false,
             "attachment": true,
-            "formInput": "file",
+            "formInput ": "file",
```
Comment (jenjou) [inline src/encoded/schemas/sc_atac_analysis_quality_metric.json]: I think a space was added accidentally here in `"formInput "`

File: src/encoded/schemas/sc_atac_analysis_quality_metric.json [hunk 2]
```diff
@@ -136,11 +139,12 @@
             }
         },
         "archr_doublet_summary_text": {
-            "title": "ArchR doublet data",
+            "title": "Pre-filtering ArchR doublet data",
```
Comment (jenjou) [inline src/encoded/schemas/sc_atac_analysis_quality_metric.json]: The property title can be kept as the original title.

File: src/encoded/schemas/sc_atac_library_complexity_quality_metric.json [hunk 3]
```diff
@@ -23,83 +23,95 @@
...
         },
         "paired_optical_duplicate_reads": {
             "title": "Paired optical duplicate reads",
+            "description": "Number of paired optical duplicates before duplicate filtering",
             "type": "integer",
             "minimum": 0
         },
         "paired_reads": {
-            "title": "Paired reads",
+            "title": "Number of paired reads before duplicate filtering",
```
Comment (jenjou) [inline src/encoded/schemas/sc_atac_library_complexity_quality_metric.json]: Title can be kept as the original

File: src/encoded/schemas/sc_atac_analysis_quality_metric.json (hunk 1 above)
Comment (khzlin) [reply src/encoded/schemas/sc_atac_analysis_quality_metric.json]: done

File: src/encoded/schemas/sc_atac_analysis_quality_metric.json (hunk 2 above)
Comment (khzlin) [reply src/encoded/schemas/sc_atac_analysis_quality_metric.json]: done

File: src/encoded/schemas/sc_atac_library_complexity_quality_metric.json (hunk 3 above)
Comment (khzlin) [reply src/encoded/schemas/sc_atac_library_complexity_quality_metric.json]: done

**Review by gabdank — DISMISSED** [review]

File: src/encoded/schemas/sc_atac_multiplet_quality_metric.json [hunk 4]
```diff
@@ -94,6 +96,7 @@
         },
         "barcodes_status": {
             "title": "Per-barcode multiplet status",
+            "description": "For each barcode, the 'IsMultiplet' field is 'True' if barcode belongs to a multiplet, 'False' if not, and 'Indeterminate' if the barcode has an insufficient fragment count (less than 500). If the barcode belongs to a multiplet, the 'PrimaryBarcode' field specifies the highest-count barcode of the multiplet ",
```
Comment (jenjou) [inline src/encoded/schemas/sc_atac_multiplet_quality_metric.json]: Can we remove the extra space at the end of this line?

File: src/encoded/schemas/sc_atac_multiplet_quality_metric.json [hunk 5]
```diff
@@ -128,6 +131,7 @@
         },
         "barcode_pairs_multiplets": {
             "title": "Multiplet barcode pair statistics",
+            "description": "Statistics for each barcode pair in a multiplet. The 'PrimaryBarcode' field specifies the highest-count barcode of the multiplet ",
```
Comment (jenjou) [inline src/encoded/schemas/sc_atac_multiplet_quality_metric.json]: Can we remove the extra space at the end of this line?

File: src/encoded/schemas/sc_atac_multiplet_quality_metric.json (hunk 4 above)
Comment (khzlin) [reply src/encoded/schemas/sc_atac_multiplet_quality_metric.json]: done

File: src/encoded/schemas/sc_atac_multiplet_quality_metric.json (hunk 5 above)
Comment (khzlin) [reply src/encoded/schemas/sc_atac_multiplet_quality_metric.json]: done

**Review by gabdank — APPROVED** [review]

**Pull request merged**